outline/data_structures.md

```diff
@@ -349,11 +349,12 @@ be confusing.
 ;=> #'clojurebridge-turtle.walk/st
 
 st
-;=> {:trinity {:x -1.7484556000744965E-6, :y 39.99999999999996, :angle
-90, :color [30 30 30]}}
+;=> {:trinity {:x -1.7484556000744965E-6, :y 39.99999999999996,
+;=>            :angle 90, :color [30 30 30]}}
```

Comment:
I read `=>` as _gives_ or _results in_, so I'd be inclined to begin wrapped output lines with `;` rather than `;=>`.